docs/coordinates/index.rst

```diff
@@ -318,13 +319,21 @@ use this option if you do not need sub-arcsecond accuracy for your coordinates::
 ..
   EXAMPLE END
 
+.. testsetup::
+
+    >>> from astropy.coordinates import EarthLocation, SkyCoord
+    >>> apo = EarthLocation(-1463969.30185172, -5166673.34223433, 3434985.71204565, unit='m')
+    >>> keck = EarthLocation(-5464487.81759887, -2492806.59108569, 2151240.19451846, unit='m')
+    >>> target = SkyCoord(10.68470833, 41.26875, unit='deg')  # M31
+
 For sites (primarily observatories) on the Earth, `astropy.coordinates` provides
 a quick way to get an `~astropy.coordinates.EarthLocation` - the
 `~astropy.coordinates.EarthLocation.of_site` method::
 
     >>> from astropy.coordinates import EarthLocation
-    >>> EarthLocation.of_site('Apache Point Observatory')  # doctest: +REMOTE_DATA +FLOAT_CMP
-    <EarthLocation (-1463969.30185172, -5166673.34223433,  3434985.71204565) m>
+    >>> apo = EarthLocation.of_site('Apache Point Observatory')  # doctest: +SKIP
+    >>> apo  # doctest: +FLOAT_CMP
```

Comment:
Is this test now pointless? Maybe. But my goal here is to minimize the number of failures when we cannot reach `sites.json` on Astropy data server in the CI and `of_site` is already tested at

https://github.com/astropy/astropy/blob/a26a6613133ea18b8ae0bfcbfd4d013bcf5adcb1/astropy/coordinates/earth.py#L341

Reply:
I think this is good for now - we just need an issue to remind us to solve this more permanently